com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -27,6 +27,12 @@ namespace MLAPI
     /// </summary>
     public abstract class NetworkedBehaviour : MonoBehaviour
     {
+        // todo: this might belong in the tick system, in the end
+        // special value to indicate "No tick information"
+        private const ushort k_NoTick = 65535;
+        // Number of ticks over which the tick number wraps back to 0
+        private const ushort k_TickPeriod = 65000;
```

Comment:
if you just want to make sure you're never as large as `k_NoTick` you could `k_NoTick - 1`

Reply:
yeah, was trying to leave some space "just in case", but this is an internal value that could change. `k_noTick - 1` would be clearer. Will do.